src/printer/index.js

```diff
@@ -638,7 +638,13 @@ function genericPrint(path, options, print) {
     }
 
     case "UnaryOp": {
-      return concat([path.call(print, "op"), path.call(print, "operand")]);
+      const separator = n.op.ast_type === "Not" ? line : "";
```

Comment:
Can `not` break without escapes or wrapping?

Reply:
Oh, good catch! Looking at this, it's actually a bit interesting because if we go to a newline and indent, we're going to be at the same starting column we were before (because `not ` is four characters, and python's indent is (typically) 4 characters). So you'd get some silly things like this:
```python
# this ends in the same place...
not my_long_var_
# ...that this ends
not \
    my_long_var_
```
That being said I'm happy to implement wrapping and indenting with a backslash, but I wonder if the "right" thing to do here  is just have the separator be a literal space in the case of a `not`.

Reply:
> just have the separator be a literal space in the case of a `not`.

Good idea 👍